fbw-common/src/wasm/systems/systems/src/shared/mod.rs

```diff
@@ -815,6 +817,71 @@ impl From<MachNumber> for f64 {
     }
 }
 
+impl MaxDifference for MachNumber {
+    fn max_diff(self, other: Self) -> f64 {
+        (f64::from(self) - f64::from(other)).abs()
+    }
+}
+
+impl MachNumber {
+    // All formulas from Jet Transport Performance Methods by Boeing (March 2009 revision)
+
+    /// Get the pressure ratio for a given pressure
+    fn delta(air_pressure: Pressure) -> f64 {
+        air_pressure.get::<hectopascal>() / 1013.25
+    }
+
+    /// Get the temperature ratio for a given temperature
+    fn theta(temperature: ThermodynamicTemperature) -> f64 {
+        temperature.get::<kelvin>() / 288.15
+    }
+
+    /// Convert the mach number to a calibrated airspeed for a given atmospheric pressure.
+    pub fn to_cas(self, air_pressure: Pressure) -> Velocity {
+        Velocity::new::<knot>(
+            1479.1
+                * ((MachNumber::delta(air_pressure)
+                    * ((0.2 * self.0.powi(2) + 1.).powf(3.5) - 1.)
+                    + 1.)
+                    .powf(1. / 3.5)
+                    - 1.)
+                    .sqrt(),
+        )
+    }
+
+    /// Convert the mach number to an equivalent airspeed for a given atmospheric pressure.
+    pub fn to_eas(self, air_pressure: Pressure) -> Velocity {
+        Velocity::new::<knot>(661.4786 * self.0 * MachNumber::delta(air_pressure).sqrt())
```

Comment:
Also here: can `661.4786` be extracted to a constant?

Reply:
Same as above.